recipes/xpack/all/conanfile.py

```diff
@@ -0,0 +1,32 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class XpackConan(ConanFile):
+    name = "xpack"
+    description = "C++ reflection, ability to convert between C++ structures and json/xml."
+    topics = ("conan", "xpack", "json", "reflection", "xml")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xyz347/xpack"
+    license = "MIT"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+       tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*", dst="include", src=os.path.join(self._source_subfolder, "include"))
```

Comment:
```suggestion
        self.copy(pattern="*", dst="include", src=self._source_subfolder))
```
seems like there is no include sub-directory, headers are within root